### PR TITLE
Adding hendersond.py

### PR DESCRIPTION
**Pull request by Hwailaluta**
hendersond.py takes a random number and returns a random string.

### CODE REVIEW

**Review by BrandonJRev — APPROVED** [review]
Good

File: hendersond.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+def hendersond (a: int) -> str:
+    '''
+    It's random my dudes.
+    '''
+    import random
+    myCoolNumbers = []
+    random.seed = a
+    for i in range(20 * a):
+        myCoolNumbers.append(random.random())
```
Comment (kylestormcloud) [inline hendersond.py:9]: Cool!

File: hendersond.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                myCoolStrings += 'i'
+            elif numero[x] == '7':
+                myCoolStrings += 'o'
+            elif numero[x] == '8':
+                myCoolStrings += 'e'
+                if random.random() > 0.5:
+                    myCoolStrings += ' '
+            elif numero[x] == '9':
+                myCoolStrings += 's'
+        myCoolStrings += ' '
```
Comment (kylestormcloud) [inline hendersond.py:42]: Coool!

File: hendersond.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        
+    myCoolOutput = ''.join(myCoolStrings)
+    return myCoolOutput
```
Comment (kylestormcloud) [inline hendersond.py:45]: Cooool!

**Review by kylestormcloud — APPROVED** [review]

**Review by srwight — APPROVED** [review]
...ok...?

**Pull request merged**